### ISSUE TITLE
paid plan required for this app?

### ISSUE THREAD

**Issue by sniktawnodnarb** [kfei/slack-cleaner#74]
Is it possible to use this app without paying for Slack?

**Comment by sgratzl** [comment]
yes sure that is original reason why this app was develop to delete old messages such that you can stick below the 10k free plan limit
